Tasks/NuGetPublisher/task.json

```diff
@@ -110,6 +110,15 @@
             "required": false,
             "helpMarkDown": "Optionally supply the path to NuGet.exe. Will override version selection.",
             "groupName": "advanced"
+        },
+        {
+            "name": "failOnEmptyNupkgMatch",
+            "type": "boolean",
+            "label": "Fail on empty nupkg match",
```

Comment:
Yes, I think we actually need to invert the option so it doesn't automatically come into effect when the tasks are updated in existing builds. Let's use the string **Continue if no packages match the "Path/Pattern to nupkg"** and make the defaultValue false. (@jmyersmsft does that make sense? my build task knowledge is rusty.)